### ISSUE TITLE
Accessibility issues - a very low score for the page stream

### ISSUE THREAD

**Issue by margondicco** [Meeds-io/meeds#1411]
Score accessibility lighthouse chrome : 63 page https://builders.meeds.io/portal/meeds/

cf attachement

Left navigation - [aria-*] attributes do not match their roles

![image](https://github.com/Meeds-io/meeds/assets/31288011/03cb9b54-7fe9-44ba-8ba0-6ecc2c56ece0)

Left navigation - 
Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.

![image](https://github.com/Meeds-io/meeds/assets/31288011/353a723b-72f2-40a9-8969-97fdb987d2c3)

Left navigation - [role]s are not contained by their required parent element

![image](https://github.com/Meeds-io/meeds/assets/31288011/50f5a19c-8b40-4d3e-b533-fdd477daece5)

Left navigation : ARIA input fields do not have accessible names

![image](https://github.com/Meeds-io/meeds/assets/31288011/c52c4e5f-07aa-4bc3-99ac-57225d2a3a6c)

Left navigation - Values assigned to role="" are not valid ARIA roles.
![image](https://github.com/Meeds-io/meeds/assets/31288011/8b8b2500-7f97-4621-a92a-3a54a4900ac2)

Porlet actions - [role]s are not contained by their required parent element

![image](https://github.com/Meeds-io/meeds/assets/31288011/8c83f426-05dc-4eea-948e-20de35e09f10)

Left navigation - stream - Buttons do not have an accessible name

and many others issues

json

[builders.meeds.io-20231205T212239.json](https://github.com/Meeds-io/meeds/files/13572541/builders.meeds.io-20231205T212239.json)
